shpc/main/settings.py

```diff
@@ -21,6 +21,14 @@
 import os
 import re
 
+# Legacy values we still support for backwards compatibility,
```

Comment:
Love this! I was just thinking "we should have a standard way to support this moving forward" and saw your implementation and was like yep that's it!